AVPlayerExample/ViewController.m

```diff
@@ -97,7 +98,7 @@ - (void)viewDidLoad {
     [self.view addGestureRecognizer:tap];
 
     // Manually configure the AudioSession
-    [self setupAudioSession];
+    [self startAudioDevice];
```

Comment:
It may make sense to separate the creation of the audio device from the starting/stopping of it.